### DISCUSSION TITLE
Blog comment on Vamana vs. HNSW - Exploring ANN algorithms Part 1

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#768] [Docs/blog comments]
# Blog comment on Vamana vs. HNSW - Exploring ANN algorithms Part 1

Vector search on disks: How does Vamana compare to HNSW?

https://weaviate.io/blog/ann-algorithms-vamana-vs-hnsw

**Comment by amityahav** [comment]
Great post!

**Comment by matchyc** [comment]
Would you share the full benchmark graph? e.g. recall >= 0.95  not just 0.99

**Reply by abdelr** [reply]
Hi, we are currently working on improving the benchmarks in these areas. We will make those public soon. In the mean time we also added Weaviate to [ann-benchmakrs](https://ann-benchmarks.com/).
